projects/mine/myproj_ideas.md

```diff
@@ -0,0 +1,34 @@
+## Project Proposals
```

Comment:
pick one for now (it can change later) and move this file there, e.g.

`projects/python/dino.md` for dinosaur taxonomy. 

Then the file can be:

```markdown
## Project Proposal - Dino Taxonomy
- dinosaur taxonomy library
     - https://github.com/junosuarez/dinosaurs is a list of dino names, but there’s no data on them
     - ‘dino’ classes, ‘clade’ classes, properties of dinos or clades, methods to compare when they lived, methods to compare their phylogenetic distance, method to pick a random dino, method to pick a random dino based on its clade
     - this is more of a database tbh


## Other Ideas

<the rest of this file>

```